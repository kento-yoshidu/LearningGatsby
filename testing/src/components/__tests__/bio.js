import React from "react"
import renderer from "react-test-renderer"

import Bio from "../bio"

describe("Bio", () => {
  it("renders corectly", () => {
    const tree = renderer
      .create(<Bio />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})