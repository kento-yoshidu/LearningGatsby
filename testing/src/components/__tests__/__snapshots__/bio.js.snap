// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders corectly 1`] = `
<div
  className="bio"
/>
`;
